ext/storage_sources/azure_store/azure_storage_source.cpp

```diff
@@ -199,21 +199,22 @@ azure_flush(WT_STORAGE_SOURCE *storage_source, WT_SESSION *session, WT_FILE_SYST
     azure_file_system *azure_fs = reinterpret_cast<azure_file_system *>(file_system);
     WT_FILE_SYSTEM *wtFileSystem = azure_fs->wt_fs;
     auto log = azure_fs->store->log.get();
+    std::string local_file_path = azure_fs->home_dir + "/" + source;
 
     WT_UNUSED(storage_source);
     WT_UNUSED(source);
     WT_UNUSED(config);
 
     // std::filesystem::canonical will throw an exception if object does not exist so
     // check if the object exists.
-    if (!std::filesystem::exists(source)) {
-        log->log_err_msg("azure_flush: Object: " + std::string(object) + " does not exist.");
+    if (!std::filesystem::exists(local_file_path)) {
+        log->log_err_msg("azure_flush: " + std::string(local_file_path) + " No such file.");
```

Comment:
We can just say `... + local_file_path + ...` now, since we're making it a `std::string` above.

Reply:
Done! [0b9a3d0](https://github.com/wiredtiger/wiredtiger/pull/8927/commits/0b9a3d065ebbea92e885c10c3934e94857660dcb)